package chpater08;

public class ExceptionEx11 {

	public static void main(String[] args) {
		throw new RuntimeException();
		//RuntimeException의 자손들은 프로그래머의 '실수'기에 예외처리 강제 X
		
	}

}
